Header header

string decision_maker
DecisionState state
RobotDecision decision